src/main/msp/msp.c

```diff
@@ -1384,6 +1384,7 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
 #ifdef USE_GPS_RESCUE
     case MSP_GPS_RESCUE:
         sbufWriteU16(dst, gpsRescueConfig()->angle);
+        sbufWriteU16(dst, gpsRescueConfig()->rescueAltitudeBufferM);
```

Comment:
This is in the wrong place. Also, it should not be added until the new parameter has been included in at least one release, to avoid churn in MSP due to changing parameters.

Reply:
Also makes sense. Will fix.